x-pack/plugins/data_usage/public/app/data_usage.tsx

```diff
@@ -42,37 +41,37 @@ export const DataUsage = () => {
     setUrlDateRangeFilter,
   } = useDataUsageMetricsUrlParams();
 
-  const [queryParams, setQueryParams] = useState<UsageMetricsRequestSchemaQueryParams>({
+  const [metricsFilters, setMetricsFilters] = useState<UsageMetricsRequestSchemaQueryParams>({
```

Comment:
note: This file is renamed in my PR. you can leave it here as it is and I'll resolve conflicts in my PR.